anchor-service/src/anchor_batch.rs

```diff
@@ -76,28 +77,42 @@ impl AnchorService {
         pin_mut!(shutdown_signal);
 
         info!("anchor service started");
+        let period =
+            TimeDelta::from_std(self.anchor_interval).expect("anchor interval should be in range");
+        let buffer = period / 12;
```

Comment:
what is buffer doing?

Reply:
its the buffer time we give ourselves to actually process the batch. I'll add a comment. For 1hour it ends up being 5 minutes.